controllers/index.js

```diff
@@ -0,0 +1,2 @@
+exports.getNonexistantEndpoint = require("./getNonexistantEndpoint.js");
```

Comment:
Not sure that we need this extra export and require step, we can just export from where the controller is created and require in from there too.